app/views/support_interface/duplicate_matches/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= render 'support_interface/candidates/candidates_navigation', title: 'Duplicate candidate matches' %>
+
+<nav class="govuk-body">
+  <% if params[:resolved] == 'true' %>
```

Comment:
Maybe assign this in the controller, now that we're using it ~twice~ three times? (once here, once in the query, once (without checking the value) below)

Reply:
Done ♻️